### DISCUSSION TITLE
Renault twingo 3 tpms light

### DISCUSSION THREAD

**Discussion by TwingoIII** [cedricp/ddt4all#1197] [Q&A] [answered]
Hello! How can I remove tpms light from renault twingo 3 dashboard? Please help!

- Hiski Eronen

**Comment by AndyIreland** [comment]
hello there - first things first , I am not sure but is your car being a Twingo3 a late registration car if it is the OBD2 port may be blocked by security gateway that you need to get unlocked by Token at the garage before any changes can be made to the cluster.

Secondly I dont know what cluster version shows up when you do a search with the magnifying glass , but here I have given this one x07 v4.1 as an example: 
![image](https://github.com/user-attachments/assets/3bb521f5-cb4c-4625-9872-fafa73080043)

**Reply by TwingoIII** [reply]
Hey! I did that but the light is still on, is there anything else I can do?

**Reply by AndyIreland** [reply]
strange would have thought it did. Did you press the prise en compte Button after you changed the settings?
![image](https://github.com/user-attachments/assets/3e112f92-5cd3-462c-bb07-2679f4d3c1f4)

and also you have disabled TPMS in other areas of ECU's like BCM/UCH ?

Are you trying to 'Disable TPMS' on your car, or are you just looking to keep TPMS on your car but just extinguish the waring lamp? - it sounds to me like you are trying to disable TPMS but i could be wrong.

**Reply by TwingoIII** [reply]
I didn't disable tpms in BCM/UHC, I will try that!

**Reply by TwingoIII** [reply]
And yeah I am trying to get rid of it.

**Reply by AndyIreland** [reply, marked as answer]
yeah you want to set this to '0' (presuming your BCM/UCH version is SW15 0 SW15 - do a scan with magnifying glass to get your correct BCM UCH for your car) 

![image](https://github.com/user-attachments/assets/41fbbe16-2f75-46f4-ba50-b9624c508267)

**Reply by TwingoIII** [reply]
Thanks! Problem solved! Thank you for the help😊

**Comment by Jeffreyvanesch** [comment]
If it does concern a Twingo 3 model (With the separate round DRL Lights)
then it can be reset by pressing the up and down buttons for a few seconds
on the right side handle.
This should clear the current warning state of the TPMS.
